docs/topics/db/aggregation.txt

```diff
@@ -643,6 +643,61 @@ aggregate that author count, referencing the annotation field:
     >>> Book.objects.annotate(num_authors=Count("authors")).aggregate(Avg("num_authors"))
     {'num_authors__avg': 1.66}
 
+.. _aggregation-mysql-only-full-group-by:
+
+Aggregating with MySQL ONLY_FULL_GROUP_BY enabled
+-------------------------------------------------
```

Comment:
```suggestion
Aggregating with MySQL ``ONLY_FULL_GROUP_BY`` enabled
-----------------------------------------------------
```

I am also tempted to move this section to the bottom of the page as it's quite niche